### PR TITLE
Adding --mandatory to distribute-release

### PR DESCRIPTION
**Pull request by yujshin**
#692

### CODE REVIEW

File: src/commands/distribute/release.ts [hunk 1]
```diff
@@ -62,6 +62,10 @@ export default class ReleaseBinaryCommand extends AppCommand {
   @longName("silent")
   public silent: boolean;
 
+  @help("Whether the release is mandatory for the testers (Not used for stores)")
```
Comment (jp-andre) [inline src/commands/distribute/release.ts]: Since this does not take in an argument, maybe reword as "make this release mandatory for testers". One way or another, make it clear that the default is `false`. :)

**Comment by karmaobserver** [comment]
For automatically build with some other services like Bitbucket for example, in "build configuration" section, there is no checkbox to distribute release as mandatory (it distributes as not mandatory version). Is there any way to make it mandatory automatically?
